Beam/resources/assets/js/components/IotaHistogram.vue

```diff
@@ -0,0 +1,93 @@
+<style lang="scss">
+.ri-histogram {
+  font-family: sans-serif;
+  font-weight: 400;
+  position: absolute;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 25%;
+  z-index: 999999;
+  transition: all 0.3s ease-in-out;
+  &__item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    color: rgb(84, 84, 84);
+    font-size: 13px;
+    background-color: #b1c5e8;
+    height: 100%;
+    margin: 2px 0;
+    padding: 5px;
+    box-sizing: content-box;
+    transition: all 0.3s ease-in-out;
+    &--max {
+      background-color: #e24242;
+      color: white;
+    }
+  }
+  &:not(:hover) {
+    width: 5px;
+    opacity: 0.5;
+    z-index: 0;
+    .ri-histogram__item {
+      width: 100% !important;
+    }
+    .ri-histogram__item__number {
+      display: none;
+    }
+  }
+}
+</style>
+
+<template>
+  <div class="ri-histogram">
+    <div
+      class="ri-histogram__item"
+      v-for="item in histogram"
+      :key="item.bucket_key"
+      :style="{width: `${calculatePercentageFromMax(item.value)}%`}"
+      :class="{'ri-histogram__item--max': item.value === maxLeavedReadersInChunk}"
```

Comment:
Please also put `ri-histogram__item` here (with a value `true` indicating that it always be used). Having both `class` and `:class` is a bit confusing when reading.

Reply:
You are right, I've merged it.